numba/cuda/cudaimpl.py

```diff
@@ -244,6 +244,57 @@ def ptx_threadfence_device(context, builder, sig, args):
     return context.get_dummy_value()
 
 
+@lower(stubs.popc, types.u4)
+def ptx_popc_u4(context, builder, sig, args):
+    """
+    LLVM IR implementation as suggested by `IntrinsicsNVVM.td`
+    """
+    return builder.ctpop(args[0])
+
+
+@lower(stubs.popc, types.u8)
+def ptx_popc_u8(context, builder, sig, args):
+    """
+    LLVM IR implementation as suggested by `IntrinsicsNVVM.td`
+    """
+    return builder.trunc(
+        builder.ctpop(args[0]),
+        Type.int(32))
```

Comment:
I think these types need to be a Numba type, not an llvmlite one? Cannot get the code to run at present as this const can't be lowered?